package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "prettier",
-  "version": "1.18.1",
+  "version": "1.19.0-pr-6157",
```

Comment:
Is this supposed to be part of the diff?

Reply:
I changed this version to make playground work.
But actually i didn't understand what version I ought to set.
Can you tell me what version should I change?
Or I need to revert this change?

Reply:
friendly ping.

Reply:
I think this line has to be reverted. I'll try and review the full PR again after I come back from vacation, unless somebody else does it before then.

Reply:
ok, i will revert it.

Reply:
@lydell 
If revert the `package.json` version, playground will not working.
Playground will read the version in `package.json` to define which options should display, but the config `since` of `--vue-indent-script-and-style` is `1.19.0`.
Should I change this version to 1.19.0 instead of revert it? 

I don't know how prettier project to control pr versions.
before merge or after merge?

Reply:
I think it's fine if the playground does not work. I guess it'll work after the next release.

Reply:
OK, I revert it.